### PR TITLE
Add policy to the SSM role to fetch from SecretsManager

### PR DESCRIPTION
**Pull request by ahinorapl**

### CODE REVIEW

File: iam.tf [hunk 1]
```diff
@@ -126,6 +126,16 @@ data "aws_iam_policy_document" "ssm_s3_cwl_access" {
...
   }
+
+  statement {
+    sid = "SecretsManagerAccessForSessionManager"
+
+    actions = [
+      "secretsmanager:GetSecretValue"
+    ]
+
+    resources = ["arn:aws:secretsmanager:*:*:secret:control/*"]
```
Comment (awarner-greshamtech) [inline iam.tf:137]: This seems quite a control specific thing to put into this module

Comment (ahinorapl) [reply iam.tf:137]: I tried to restrict it as much as possible so that I don't give too much access to the resources

Comment (awarner-greshamtech) [reply iam.tf:137]: I see that but it does mean this is no longer a reusable module

**Comment by ahinorapl** [comment]
moving this to the environment-specific policy in the account setup

**Pull request closed without merging**